### ISSUE TITLE
feature request: imageblock streamfield, documentblock

### ISSUE THREAD

**Issue by onno-timmerman** [wagtail/wagtail-localize#281]
Currently in streamfields the imageblocks and documentblock are not supported. They give only a "number".

**Comment by codingcatgirl** [comment]
I saw your other bug report already, and now i'm wondering if you are using the same version as me. Because for me, it does not display a number for imageblocks.

**Comment by iankameron** [comment]
I am maybe seeing something similar, but only when the imagechooserblock is in a StreamBlock that's nested into another StreamField. If this is a separate issue, I'll log separately.

in original language page
![image](https://user-images.githubusercontent.com/38037829/104887616-ecfb3600-59ae-11eb-8a33-6ea3298a2de6.png)

translation page
![image](https://user-images.githubusercontent.com/38037829/104887648-f97f8e80-59ae-11eb-8b47-ff0b4d52cd04.png)

imagechooserblock showing up as expected when it's just in a simple StreamField (allowing selection of a different image for the translated Page)
![image](https://user-images.githubusercontent.com/38037829/104887866-51b69080-59af-11eb-8d48-9bcff38c6806.png)

Page definition
```
    body = StreamField(
        block_types=[
            ('section', SectionBlock()),
        ]
    )
```

Where SectionBlock defined as:
```
class SectionBlock(blocks.StructBlock):
    section_title = blocks.CharBlock(icon='title')
    body = ContentBlock()
    code = CodeSet()
```

And ContentBlock:
```
class ContentBlock(blocks.StreamBlock):
    section_text = blocks.RichTextBlock()
    image = ImageChooserBlock()
```

**Comment by zerolab** [comment]
This was fixed in #480 and working as of version 1.0
